src/main/java/Command/MarkCommand.java

```diff
@@ -0,0 +1,57 @@
+package Command;
+import Duke.DukeException;
+import Duke.DukeUi;
+import Duke.Storage;
+import Duke.Task;
+import Duke.TaskList;
+import java.io.IOException;
+
+/**
+* Command that marks a specified task as complete when executed.
+*/
+public class MarkCommand extends Command {
+
+    boolean exit;
+    private String userAction;
+
+    public MarkCommand(String userAction) {
+        this.exit = false;
+        this.userAction = userAction;
+    }
+
+    public boolean isNumeric(String str) {
```

Comment:
perhaps you can put isNumeric in one of the classes and use it, so that you do not have to initialise it everytime in a new class :)